pkg/proto/operator/v1/operator_grpc.pb.go

```diff
@@ -4,10 +4,10 @@ package operator
 
 import (
 	context "context"
-	empty "github.com/golang/protobuf/ptypes/empty"
 	grpc "google.golang.org/grpc"
 	codes "google.golang.org/grpc/codes"
 	status "google.golang.org/grpc/status"
+	emptypb "google.golang.org/protobuf/types/known/emptypb"
```

Comment:
Why all these changes in generated proto? Are you using a different version of code generator to generate the code?